### DISCUSSION TITLE
[operator] Let static k8s resources be generated from the Helm chart's default values

### DISCUSSION THREAD

**Discussion by consideRatio** [aquasecurity/starboard#211] [Development]
In order to not maintain two different sets of Kubernetes resources, I suggest to let the starboard-operator Helm chart render static resource that can be referenced. What can otherwise happen, is that contributors end up fixing issues in the Helm chart but not in the static resource files etc, and things go out of sync.

**Comment by danielpacak** [comment]
I fully agree that it doesn't make sense to maintain both. Once this [issue](https://github.com/aquasecurity/starboard/issues/187) is resolved we'll get rid of static YAML manifest and suggest using `helm template ./deploy/helm` command or `helm install`.

**Reply by danielpacak** [reply]
Let's mark this discussion as resolved and follow up on #216
